CHANGES.txt

```diff
@@ -31,6 +31,11 @@ Breaking Changes (Packaging only)
 Changes
 =======
 
+ - Added new HBA setting ``ssl`` which allows to control user authentication
+   based on the connection scheme used for the ``pg`` protocol.
```

Comment:
i think "connection scheme" is misleading. actually i would just leave out the first paragraph, since this is a consequence out of the change described in the second.